README.md

```diff
@@ -867,6 +870,14 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
     recoveryToken: 'x0whAcR02i0leKtWMZVc'
     ```
 
+## MFA Challenge flow without sign out link
```

Comment:
Hmm, good thing we added the doc - looking at this now, would prefer to add this under our feature settings rather than a top level config item.